src/Sharpliner/AzureDevOps/IfCondition.cs

```diff
@@ -7,11 +7,17 @@ namespace Sharpliner.AzureDevOps;
 /// </summary>
 public abstract class IfCondition : Condition
 {
+    internal bool IsElseIf = false;
```

Comment:
Can you not make this a boolean flag but instead use inheritance and possibly override some of the fields such as `TagStart`?
Can you do the same for the builder?

Reply:
@jr-krandino how about this one?